tests/test_imod_coupler/test_ribametamod_cases.py

```diff
@@ -326,6 +328,59 @@ def case_bucket_model(
     )
 
 
+def case_bucket_model_no_subgrid(
```

Comment:
Is the case of basins coupled to metaswap but not to modflow also tested here  ?

Reply:
No that is indeed not tested for now.